packages/webdriverio/src/commands/element/isDisplayed.js

```diff
@@ -34,20 +34,46 @@
  * </example>
  *
  * @alias element.isDisplayed
- * @return {Boolean} true if element(s)* [is|are] displayed
+ * @return {Boolean} true if element is displayed
  * @uses protocol/elements, protocol/elementIdDisplayed
  * @type state
  *
  */
 
+import { ELEMENT_KEY } from '../../constants'
+import { getBrowserObject } from '../../utils'
+import isElementDisplayedScript from '../../scripts/isElementDisplayed'
+
+const noW3CEndpoint = ['MicrosoftEdge', 'safari']
```

Comment:
What if people write this browser name with lower case? or just write `edge`. Are the different ways to define edge as a browser in the capabilities. What we definitely need to do though is to lowercase this.

Reply:
Done.